libs/isr/package.json

```diff
@@ -6,6 +6,7 @@
   "peerDependencies": {
     "@angular/common": "^17.0.0",
     "@angular/core": "^17.0.0",
+    "@angular/ssr": "^17.0.0",
     "@types/node": "^18.0.0 || ^20.0.0",
```

Comment:
Update the lockfile
https://github.com/rx-angular/rx-angular/actions/runs/7619336794/job/20861381969#step:4:166